rust/src/services/messages/state_machine.rs

```diff
@@ -23,9 +23,11 @@ use crate::{
     PetError,
 };
 
-/// The request type handled by [`StateMachineService`]
+/// [`StateMachineService`] request type
 #[derive(Debug, From)]
 pub struct StateMachineRequest(MessageOwned);
+
+/// `[StateMachineService`] response type
 pub type StateMachineResponse = Result<(), StateMachineError>;
 
 #[derive(Debug, Error)]
```

Comment:
the `StateMachineError` is missing documentation.

Reply:
Added.